index.js

```diff
@@ -124,7 +124,13 @@ UglifyWriter.prototype.processFile = function(inFile, outFile, relativePath, out
     debug('[finsihed]: %s %dKB in %dms', relativePath, (result.code.length / 1000), total);
 
     if (total > 20000) {
-      console.warn('[WARN] `' + relativePath + '` took: ' + total + 'ms (more then 20,000ms)');
+      console.warn(
+        '[WARN] `' + relativePath + '` took: ' + total + 'ms (more then 20,000ms). ' +
+        'This is likely due to the total amount of code that is being minified. You may be ' +
```

Comment:
as this may be printed multiple times. Can i recommend simply adding a shortened `git.io` url?

Pointing to https://github.com/ef4/broccoli-uglify-sourcemap#slow-rebuilds ?